### PR TITLE
Added random forest model with cv results

### PR DESCRIPTION
**Pull request by YXIN15**

### CODE REVIEW

File: src/models/chocolate_rand_forest.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+from docopt import docopt
+from scipy.stats import randint
+from sklearn.pipeline import make_pipeline
+from sklearn.ensemble import RandomForestRegressor
+
+from ..preprocessor.chocolate import preprocessor
+from .base_chocolate_model_tuner import BaseChocolateModelTuner
+
+
+class ChocolateDecisionTreeTuner(BaseChocolateModelTuner):
```
Comment (netsgnut) [inline src/models/chocolate_rand_forest.py]: Consider renaming this tuner class as, say, `ChocolateRandForestTuner`?

File: src/models/chocolate_rand_forest.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        return super().param_distribution() | {
+            "randomforestregressor__n_estimators": randint(low=50, high=1000),
+            "randomforestregressor__max_depth": randint(low=10, high=100)
+        }
+
+
+opt = docopt(__doc__)
+
+if __name__ == "__main__":
+    tuner = ChocolateDecisionTreeTuner()
```
Comment (netsgnut) [inline src/models/chocolate_rand_forest.py]: Remember to update this line too, if you have renamed the class :+1:

File: src/models/chocolate_rand_forest.py (hunk 1 above)
Comment (YXIN15) [reply src/models/chocolate_rand_forest.py]: Good catch! Will fix that.

File: src/models/chocolate_rand_forest.py (hunk 2 above)
Comment (YXIN15) [reply src/models/chocolate_rand_forest.py]: Thanks Kelvin!

**Review by netsgnut — APPROVED** [review]

**Pull request merged**